lua/starfall/libs_sv/entities.lua

```diff
@@ -1104,6 +1104,43 @@ function ents_methods:setPhysicsUpdateListener(func)
 	end
 end
 
+--- Marks an entity as a trigger, setting a callback function to run whenever other entities enter or leave the bounds of the first.
+--- The entity will still invoke the callback even if not solid and no physical collision occurs, unlike Entity:addCollisionListener.
+--- The callback function arguments are: (Entity object, bool entering) (entering is true if the object is entering the bounds, false if leaving)
+--- https://developer.valvesoftware.com/wiki/Triggers
+---
+--- You can only use this function on these classes:
+--- - starfall_prop
+--- - starfall_processor
+-- @param function|nil func The callback function. Use nil to remove an existing callback.
+function ents_methods:setTriggerListener(func)
```

Comment:
Would be better to have two function input args for startTouch and endTouch instead of using a boolean. Code looks fine, but I will test it soon.